data-prepper-api/src/test/java/com/amazon/dataprepper/model/configuration/PipelinesDataFlowModelTest.java

```diff
@@ -39,9 +40,9 @@ void setup() {
     void testSerializing_PipelinesDataFlowModel_empty_Plugins_with_nonEmpty_delay_and_workers() throws JsonProcessingException {
         String pipelineName = "test-pipeline";
 
-        final PluginModel source = new PluginModel("testSource", null);
-        final List<PluginModel> preppers = Collections.singletonList(new PluginModel("testPrepper", null));
-        final List<PluginModel> sinks = Collections.singletonList(new PluginModel("testSink", null));
+        final PluginModel source = new PluginModel("testSource", (Map<String, Object>) null);
```

Comment:
We could create a constructor without pluginSettings if we wish to support it. This way we can avoid casting and passing a null value through.

```
    public PluginModel(final String pluginName) {
        this(pluginName, new InternalJsonModel());
    }
 ```